src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -302,7 +302,7 @@ public event System.EventHandler? ProcessExit { add { } remove { } }
         public event System.ResolveEventHandler? ReflectionOnlyAssemblyResolve { add { } remove { } }
         public event System.ResolveEventHandler? ResourceResolve { add { } remove { } }
         public event System.ResolveEventHandler? TypeResolve { add { } remove { } }
-        public event System.UnhandledExceptionEventHandler? UnhandledException { add { } remove { } }
+        public event Func<Exception, bool>? UnhandledException { add { } remove { } }
```

Comment:
This is old pre-existing API. It should say as-is.

Reply:
Looks like you edited wrong place in the file.

Reply:
Oops, fixed. Thanks!